Sources/SwiftLayout/Extension/Anchors+Convenience.swift

```diff
@@ -123,9 +129,56 @@ extension Anchors {
     ///  - offset: CGFloat. top constant is `offset`, bottom constant is `-offset`, trailing constant is `-offset`
     ///
     /// - Returns: ``Constraint``
-    @AnchorsBuilder
-    public static func shoe(offset: CGFloat = .zero) -> SwiftLayout.Constraint {
-        horizontal(offset: offset)
-        Anchors(.bottom).equalTo(constant: -offset)
+    public static func shoe(offset: CGFloat = .zero) -> Anchors {
+        let horizontal = horizontal(offset: offset)
+        let bottom = Anchors(.bottom).equalTo(constant: -offset)
+        return horizontal.union(bottom)
+    }
+    
+    /// ``Anchors`` for width, height toward toItem: ``ConstraintableItem``
+    ///
+    ///
+    /// - Parameters:
+    ///  - toItem: constraint second item, ``ConstraintableItem``
+    ///  - length: constant
+    ///
+    /// - Returns: ``Constraint``
+    public static func size<I: ConstraintableItem>(_ toItem: I, length: CGFloat = .zero) -> Anchors {
```

Comment:
I think, "offset" will be better than "length" in this method

Reply:
oh you right, offset is more fit in this situation.